### ISSUE TITLE
Error: Cannot find module './eslintrc.json'

### ISSUE THREAD

**Issue by andrewn** [johnie/eslint-config-airbnb-flow#4]
Steps to reproduce:
1. `npm install eslint-config-airbnb-flow --save-dev`
2. Add `{ "extends" : ["airbnb", "airbnb-flow"] }` to `.eslintrc`
3. Run `eslint somejavascript.js`

Stack trace:

```
$ eslint somejavascript.js 
Cannot read config file: /Users/andrew/Projects/embedded-react-native/node_modules/eslint-config-airbnb-flow/index.js
Error: Cannot find module './eslintrc.json'
Referenced from: /Users/andrew/Projects/embedded-react-native/.eslintrc
Error: Cannot read config file: /Users/andrew/Projects/embedded-react-native/node_modules/eslint-config-airbnb-flow/index.js
Error: Cannot find module './eslintrc.json'
Referenced from: /Users/andrew/Projects/embedded-react-native/.eslintrc
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/andrew/Projects/embedded-react-native/node_modules/eslint-config-airbnb-flow/index.js:1:98)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
```

I can confirm that `eslintrc.json` is not in the installed NPM package in node_modules.

**Comment by johnie** [comment]
@andrewn You're absolutely right. Thank you for noticing. This is now fixed! Please update your package to `1.0.2`

**Comment by andrewn** [comment]
Works great, thanks
